packages/nextjs/src/copy-config-files.ts

```diff
@@ -5,18 +5,57 @@ const fs = require('fs')
 
 const debug = process.env.HONEYBADGER_DEBUG === 'true'
 
+function usesTypescript() {
+  return fs.existsSync('tsconfig.json')
+}
+
+function usesAppRouter() {
```

Comment:
I saw in the Next documentation that they're supporting `app` and `pages` concurrently.. is that going to cause any issues ?

Reply:
That's a nice catch! I will have to handle that case as well 😄